### ISSUE TITLE
ETL Course: Feedback Step 1 + 2

### ISSUE THREAD

**Issue by PRijnbeek** [EHDEN/Academy#19]
Find below some feedback. Happy to discuss in a call or through the issues tracker.

**Welcome Text**
- [x] We refer to a part 2 but it is not clear what this is about. Do we still need that? I suggest we add that this course is based on the course given at the European OHDSI Symposium etc with a link to the symposium. 
- [x] I would make the image much smaller. 
- [x] Eventually we need to add the other teachers. @clairblacketer could you add a picture to your profile to test this shows up.

**Forum**
- [x] Has completed check box below it. Will this be activated once they have posted something? Not a bad thing probably? We should mention somewhere how we want them to use the forum for this course?
- [x] Is there an option to make sections in forum per topic.

**Introduction**
- [x] "Describes the choice behind the OMOP CDM" sounds a bit strange to me. I suggest to also refer to the Setup for lesson (intoduction see point below) in the intro text
- [ ] Why CDM: This is slide deck that is also discuss in the video's correct (if not we could keep it as background material / refresher but there is the other course on this). We need to explain to the students how to use this material somewhere. I suggest we create a course introduction section as we did in the CDM one (https://www.test-academy.ehden.eu/mod/page/view.php?id=22). 
- [ ] Setup for the lesson: This should point to the other course from Amazon. We could put this in the course instruction section. I think the current lecture itself (the video) is doing much more than setup it is more an "Introduction to the course". Is there a short Welcome part in the video besides this, if not this is fine but we have to rename the section. Make the Lauren part a separate video together with Synthia part

**ETL - Step 1**
- [ ] White Rabbit: it would be nice if we first have an intro (landing) page referring to the Github and section in The Book of OHDSI etc. Video Lecture and Self-guided (we need a better name for this) can be different steps in the lecture, I think it is useful they do both not choose (first video, the guided version?). The links to the OHDSI tutorial github I would add to the course instruction page. In the lecture it explains that the chrome on the box has links which is nice etc, this is information we need to have somewhere as well. 

- [ ] White Rabbit Assignment: they need to upload, we then have to grade that. We need to have the right answer to the approve it. What if it is wrong we provide feedback to the student directly through message in the academy or is there a better way to do this?

- [ ] Rabbit in a hat: see comment earlier about the landing page and then video + self guided. Video does not work for me (no page showing up). 

- [ ] Rabbit in a hat assignment; I would cutoff the few second where Erica says you will show the answer since we will not.

**ETL - Step 2**
- [ ] suggest to change the text "People with medical knowledge create .." and replace this by a more clear intro text.
- [ ] Skimmed through the video and saw at some point a 15min clock is that the exercise we have later?

**Comment by clairblacketer** [comment]
Thanks @PRijnbeek! I will work on addressing these

**Comment by clairblacketer** [comment]
I have checked off the ones above that were pretty straightforward and I have completed the following:

**Forum**
* I removed the "completion" check box as it was unnecessary and should reduce confusion.
* I am not sure how it will look from the student side and if it will allow for multiple topics going at once.

**Introduction**
* I have added an introduction to the course but I did not add the 'Why CDM' video. I would like to chat about the best place for that video.

**ETL Step - 1**
* I added a landing page and changed the content so that it is video and then self-guided for both WhiteRabbit and Rabbit-In-A-Hat
* I did some testing with the grading and I think the best way will be feedback through the site
* Fixed the Rabbit-In-A-Hat video link

**ETL Step - 2**
* Changed the description
